chunk/storage_client.go

```diff
@@ -18,6 +18,10 @@ type StorageClient interface {
 	CreateTable(name string, readCapacity, writeCapacity int64) error
```

Comment:
I should really stick a ctx on the front of this interface and roll the instrumentation into the implementation...